src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  /* eslint-enable */
```

Comment:
- [ ] As pointed above, let's fix them instead of disabling them. 
- If you have Linters issues while trying to make the component class-based, I'll strongly advise you to try out React pure component. It will be fun!